src/styles/product__card.scss

```
@@ -72,27 +82,29 @@
     font-size: 16px;
     line-height: 19px;
     text-align: right;
-    margin: 0 0 20px;
+    margin: 0;
   }
+}
 
-  &__link {
-    width: 170px;
-    height: 40px;
-    line-height: 40px;
-    font-size: 14px;
-    font-weight: $font-weight;
-    text-align: center;
-    text-decoration: none;
-    text-transform: uppercase;
-    background-color: $color;
-    border-radius: $border-radius;
-    color: white;
-  }
+.product__link {
```

Я би перемістив весь цей елемент всередину .product { } і назвав
```suggestion
&__link {
```

---

done